tests/std/tests/GH_001912_random_distribution_operator_const/test.compile.pass.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _ALLOW_RANDOM_DISTRIBUTION_CONST_OPERATOR
+
+#include <random>
+
+using namespace std;
+
+template <typename Dist>
+void test() {
+    default_random_engine random;
+    const Dist dist;
+    const auto res1 = dist(random);
+    (void) res1;

```

Comment:
Why store the result _and_ silence "unused variable" warnings instead of simply discarding a la `(void) dist(meow)`? (Pure style comment, feel free to ignore.)